### DISCUSSION TITLE
Reporting BISs Support in Oxalis

### DISCUSSION THREAD

**Discussion by aaron-kumar** [OxalisCommunity/oxalis#611] [Polls]
You must have noticed new reporting requirement from OpenPeppol. This require AP service provider to send  "[Transaction Statistics Reporting](https://test-docs.peppol.eu/reporting/tsr/)" and [Peppol End User Reporting](https://test-docs.peppol.eu/reporting/eur/) to Peppol AP.

Reporting BIS was initially published on 25 April 2022 and was open for 2 months review which end on 25 June 2022. This timeline is based on new [Internal Regulations](https://openpeppol.atlassian.net/wiki/download/attachments/2756772071/Internal%20Regulations%20-%20Use%20of%20Peppol%20Network_v1.0%20APPROVED%202021.11.16.pdf?api=v2). 
Based on these internal regulations, minimum time that must be allowed for implementation (migration) is 6 months (_Maximum time is Not specified_). That means this should be available as early as end of 2022 (_if we go by Minimum time guidelines_). Though there is lot of discussion happening within OpenPeppol and SP provider community about it. It is very likely that there could be some changes in reporting types required in initial phase or some changes in timeline. 

"*" - condition apply regarding availability of functionality 

**Note**: End User reporting is currently out of scope of Oxalis due to scope and purpose of software

**Comment by PPotappel** [comment]
Easier access to the AP-identifier in the REM would be enough for us.
